aspnetcore-webapi/UsingResultPatternInNETWebAPI/UsingResultPatternInNETWebAPI/v4/TheResultPattern/CustomError.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace UsingResultPatternInNETWebAPI.v4.TheResultPattern;
+
+public sealed record CustomError(string Code, string Message)
+{
+    private static readonly string NotFoundCode = "NotFound";
+    private static readonly string ValidationErrorCode = "ValidationError";
+
+    public static readonly CustomError None = new(string.Empty, string.Empty);
+
+    public static CustomError NotFound(Guid id)
```

Comment:
Same comments here, it can accept two parameter, one for propertName and another for value. The error message can use those parameter to be dynamic.